third_party/blink/renderer/modules/cobalt/h5vcc_metrics/metrics_event.idl

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A custom event that carries deep link information. This event is
+// dispatched on the H5vccRuntime object when a deep link is received.
+[
+    Exposed=Window,
+    SecureContext
+]
+interface MetricsEvent : Event {
+    readonly attribute DOMString tbd;
```

Comment:
Is tbd a shorthand for something relevant I don't know? Or do you mean to replace it with a better name?

Reply:
It's indeed "To Be Defined", but likely it'll still be a DOMString.